apps/root/src/pages/earn/home/components/wizard/components/wizard-selection/index.tsx

```diff
@@ -35,11 +35,23 @@ const StyledTitle = styled(Typography).attrs({ variant: 'h5Bold' })`
   `}
 `;
 
-const onRenderValue = (defaultText: string, option?: RewardSelectorOption | AssetSelectorOption) => (
-  <Typography variant="h5Bold" color="primary">
-    {option ? option.token.symbol : defaultText}
-  </Typography>
-);
+const RenderWizardItem = ({
+  item,
+  isRenderValue,
+}: {
+  item: RewardSelectorOption | AssetSelectorOption;
+  isRenderValue?: boolean;
```

Comment:
What does isRenderValue mean? that is the value that is selected currently?

Reply:
Yes, is the selected item, but also rendered when the dropdown is closed